src/pkg/cli/configDelete_test.go

```diff
@@ -0,0 +1,37 @@
+package cli
+
+import (
+	"context"
+	"testing"
+
+	"github.com/DefangLabs/defang/src/pkg/cli/client"
+	defangv1 "github.com/DefangLabs/defang/src/protos/io/defang/v1"
+)
+
+func TestConfigDelete(t *testing.T) {
+	ctx := context.Background()
+	provider := MockConfigDeleteProvider{}
+
+	t.Run("expect no error", func(t *testing.T) {
+		if err := ConfigDelete(ctx, "test", provider, "test_name"); err != nil {
+			t.Fatalf("ConfigDelete() error = %v", err)
+		}
+	})
+
+	t.Run("expect error on DryRun", func(t *testing.T) {
+		DoDryRun = true
+		defer func() { DoDryRun = false }()
```

Comment:
Use `t.Cleanup` in tests instead of `defer` so that we can ensure it will run even if `t.Fatal` is called.

```suggestion
		t.Cleanup(func() { DoDryRun = false })
```